packages/earley/earley.2.0.0/opam

```diff
@@ -0,0 +1,32 @@
+synopsis: "Parsing library based on Earley Algorithm"
+description:
+"""
+Earley is a parser combinator library base on Earley's algorithm. It
+is intended to be used in conjunction with an OCaml syntax extension
+which allows the definition of parsers inside the language. There is
+also support for writing OCaml syntax extensions in a camlp4 style.
+"""
+
+opam-version: "2.0"
+maintainer: "Rodolphe Lepigre <rodolphe.lepigre@inria.fr>"
+bug-reports: "https://github.com/rlepigre/ocaml-earley/issues"
+homepage: "https://github.com/rlepigre/ocaml-earley"
+dev-repo: "git+https://github.com/rlepigre/ocaml-earley.git"
+authors: [
+  "Christophe Raffalli <christophe@raffalli.eu>"
+  "Rodolphe Lepigre <rodolphe.lepigre@inria.fr>" ]
+license: "CeCILL-B_V1"
+doc: "https://rlepigre.github.io/ocaml-earley/"
+
+depends: [
+  "ocaml" { >= "4.03.0" & <= "4.07.1" }
```

Comment:
Why is it constrained for 4.08? If there is a good reason for this could you at least have the upper-bound to `< "4.08"` instead? It would probably make more sense if there is a new security release 4.07.2 or something

Reply:
I know this is a bit weird, but I don't really have a choice here. We need to generate version-specific code, and we also rely on `compiler_libs` code that is marked unstable (changes may occur between minor versions, this append before.).